resources/js/src/app/mixins/url.ts

```diff
@@ -32,6 +32,16 @@ export default class Url extends Vue {
         return {};
     }
 
+    sanitizeHTML(str : string): string {
+
+        const map = {
+            '<': '&lt;',
+            '"': '&quot;',
+            '\'': '&#039;',
+        };
+        return str.replace(/[&<"']/g, function(m) { return map[m]; });
```

Comment:
⚙️  To be more secure please add more characters to the filter that in the end this characters will filtered:

```
&    &amp;
<    &lt;
>    &gt;
"    &quot;
'    &#x27;
```

[Source](https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html)